tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -0,0 +1,176 @@
+import copy
+import math
+import pickle
+from threading import Thread
+
+import boto3
+
+from tron.serialize.runstate.shelvestore import ShelveKey
```

Comment:
do we really need to involve shelve keys here? can we just make this an implementation detail of dynamodb store itself? like have `build_key` simply  return `f"{type}:{iden}"`? i'd like this class to not depend on anything but boto.

Reply:
It's easier to implement in this way. DynamoDB and ShelveStateStore could thus share the same key. Once thing I can do is to convert ShelveKey to f"{type}:{iden}" before storing to DynamoDB and reading. Then, after migration, I can just use  f"{type}:{iden}".